packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -193,6 +193,8 @@ export const ChatScreen = () => {
   const composeRef = useRef<View | null>(null)
   const chatContainerRef = useRef<View | null>(null)
   const messageTop = useRef(0)
+  const messageWidth = useRef(0)
+  const messageHeight = useRef(0)
```

Comment:
Could combine into a single `messageDimensions` ref with `width` and `height`.

Reply:
actually no longer need width